apps/api-documenter/src/documenters/DocumenterConfig.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import * as path from 'path';
+import { JsonSchema, JsonFile } from '@microsoft/node-core-library';
+import { IConfigFile } from './IConfigFile';
+
+/**
+ * Helper for loading the api-documenter.json file format.  Later when the schema is more mature,
+ * this class will be used to represent the validated and normalized configuration, whereas `IConfigFile`
+ * represents the raw JSON file structure.
+ */
+export class DocumenterConfig {
+  /**
+   * The JSON Schema for API Extractor config file (api-extractor.schema.json).
+   */
+  public static readonly jsonSchema: JsonSchema = JsonSchema.fromFile(
+    path.join(__dirname, '../schemas/api-documenter.schema.json'));
```

Comment:
>../ [](start = 26, length = 3)

Change this call to `path.resolve(__dirname, '..', 'schemas', 'api-documenter.schema.json'));

Reply:
Changed